src/serial-console/azext_serialconsole/custom.py

```diff
@@ -695,3 +671,94 @@ def enable_serialconsole(cmd):
 def disable_serialconsole(cmd):
     client = cf_serialconsole(cmd.cli_ctx)
     return client.disable_console()
+
+
+def get_region_from_storage_account(cli_ctx, resource_group_name, vm_vmss_name, vmss_instanceid):
+    from azext_serialconsole._client_factory import storage_client_factory
+    from knack.log import get_logger
+
+    logger = get_logger(__name__)
```

Comment:
Usually, we will put these lines at the beginning of custome.py file.

Reply:
I put that line there because really only that function cares about that import instead of the whole file.  The import is only used in that method and instantiated at that time was my thought process behind that change.

Reply:
Do you have the access to merge this PR because I don't.